lib/deployment_runner.py

```diff
@@ -83,8 +88,11 @@ def rhv_configuration(self, page):
         '''Configuration'''
         page.set_root_passwords(self.password)
         page.set_engine_passwords(self.password)
-        page.set_data_center_name(self.rhv.data_center_name)
-        page.set_cluster_name(self.rhv.cluster_name)
+
+        if self.rhv.rhv_setup_type != 'selfhost':
```

Comment:
Maybe we should leave a comment with the bugzilla that had these not be settable for selfhosted.